.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: python
+python:
+  - "2.7"
+  - "3.4"
+  - "3.5"
+  - "3.6"
+  - "3.7"
+  - "3.8"
+install: pip install -r requirements-dev.txt codecov
+script: tox
+after_success:
+  - codecov
```

Comment:
based on my previous comment, maybe we could simplify this and get rid of python 2.7?
I never used travis with tox, do we need sth here extra to make cloudinary tests passing?

Reply:
To use tox in travis simply add credentials as environment variables in project settings in travis